pandas/core/groupby/grouper.py

```diff
@@ -693,89 +692,9 @@ def indices(self) -> dict[Hashable, npt.NDArray[np.intp]]:
     def codes(self) -> npt.NDArray[np.signedinteger]:
         return self._codes_and_uniques[0]
 
-    @cache_readonly
-    def _group_arraylike(self) -> ArrayLike:
```

Comment:
nice! i think this should have been made unnecessary a while back when Index was made able to hold arbitrary EAs